c2corg_ui/static/js/auth/auth.js

```diff
@@ -55,6 +55,12 @@ app.AuthController = function($scope, appApi, appAuthentication,
    */
   this.scope_ = $scope;
 
+  if (!this.scope_['login']) {
+    this.scope_['login'] = {};
+  }
```

Comment:
I think @gberaudo solution is OK since we set "login" in the scope with the default values (remember: true) before actually setting the username/password from the form.